exporter/sentryexporter/sentry_exporter.go

```diff
@@ -42,6 +42,8 @@ var canonicalCodes = [...]sentry.SpanStatus{
 	sentry.SpanStatusOK,
 	sentry.SpanStatusUnknown,
 }
+// exceptionEvents is a mutable slice of exception events in spans recorded as Sentry events.
+var exceptionEvents []*sentry.Event
```

Comment:
This shouldn’t be a global variable, it should be in the `pushTraceData` func.

Reply:
Okay, will change.